packages/docs-site/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "module": "NodeNext",
+    "allowJs": true,
+    "strict": true,
+    "jsx": "preserve"
+  },
+  "include": ["./.vitepress/**/*"]
```

Comment:
Doesn't this mean stuff in `packages/docs-site/` but outside `packages/docs-site/.vitepress/` doesn't get typechecked?